app/screens/phone-auth-screen/phone-input.tsx

```diff
@@ -138,9 +138,15 @@ export const PhoneInputScreen: React.FC = () => {
 
   useEffect(() => {
     if (phoneNumber) {
+      if (phoneNumberIsLocalTestAccount(phoneNumber)) {
```

Comment:
Could you do `if (galoyInstance.name === 'Local')` or am I missing something?

Reply:
oh yup indeed. will do that.